wraith/bin/wraith.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+read -p "Specify the port number that your local dev environment is running on: " port
+
+port=${port:-5000}
```

Comment:
I don't think you need to even prompt for this. Check if `PORT` is set or default to `3000`.